support-frontend/assets/components/headers/header/header.scss

```diff
@@ -318,3 +318,12 @@ $header-links-min-width: gu-span(12);
     display: none;
   }
 }
+
+.component-header-links__li.component-header-links__li--show-on-tablet {
```

Comment:
I'm not totally up on BEM, but I think "show on tablet" is a utility class rather than a modifier?

Reply:
The idea is to have this locally scoped to the header links, otherwise I would agree